teams/helpers.py

```diff
@@ -76,7 +76,7 @@ def group_participants(participants, num_teams):
         participant_groups.setdefault(participant_level, [])
         participant_groups[participant_level].append({
                 'userid': participant.id,
-                'name': participant.username,
+                'name': participant.email,
```

Comment:
If the field is `name` when why are we using `participant.email` instead of `participant.username`? Should we rename this key to `email: participant.email` in that case?

Reply:
Actually you are right. I changed the value to be `slack_display_name` and default to `email` if not available, but leave it at `name`.